scripts/run-once-scripts/fix-null-page-counts.ts

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env -S npx ts-node --transpile-only
+
+import { createApplicationContext } from '@web-api/applicationContext';
+import { getDbReader, getDbWriter } from '@web-api/database';
+import { settlePromises } from '@web-api/utilities/settlePromises';
+import { sql } from 'kysely';
+import pLimit from 'p-limit';
+import {
+  parseArgsAndEnvVars,
+  ScriptConfig,
+} from 'scripts/helpers/parseArgsAndEnvVars';
+
+const scriptConfig: ScriptConfig = {
+  description:
+    'Fixes issues with docket entries that do not have the number of pages set',
+  environment: {
+    env: 'ENV',
+  },
+  requireActiveAwsSession: true,
+};
+
+parseArgsAndEnvVars(scriptConfig);
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+  console.log('Starting...');
+  const applicationContext = createApplicationContext({});
```

Comment:
At one point there was a push to import `applicationContext` directly in instances like these, where it is being instantiated with an empty object in a script, instead of using `createApplicationContext`. I believe this is because you wind up with the same static result either way and calling `createApplicationContext` does that same work all over again.

Just an FYI for future reference, though; not worth changing in a script that will only run once.